library/L2_HAL/sensors/environment/light/temt6000x01.hpp

```diff
@@ -13,36 +13,32 @@ class Temt6000x01 final : public LightSensor
 {
  public:
   /// @param adc The ADC peripheral used to read the sensor data.
-  /// @param adc_reference_voltage The reference voltage of the ADC. This value
-  ///                              will vary based on the chosen MCU.
+  /// @param reference_voltage The voltage supplied to this device.
   /// @param pull_down_resistance The resistance of the pull down resistor used
   ///                             in the circuit.
   explicit constexpr Temt6000x01(sjsu::Adc & adc,
-                                 units::voltage::volt_t adc_reference_voltage,
+                                 units::voltage::volt_t reference_voltage,
```

Comment:
A better name may be `supply_voltage`

Reply:
Would the supply voltage be necessary since it's never used?

Reply:
Well yes because bit should be used. Supply voltage should be used in this driver

Reply:
Realized that the supply voltage is not necessary for this driver whats so ever and the current can be derived without knowing the circuits supply voltage.